src/components/text-block.tsx

```diff
@@ -5,5 +5,5 @@ export const TextBlock = styled("div")`
     text-align: justify;
     font-size: ${fontSizes[1]};
     max-width: ${breakPoints[3]};
-    margin: ${spacing[2]} 0;
+    margin: 0 auto ${spacing[2]} auto;
```

Comment:
this can be written as `margin: 0 auto ${spacing[2]}` and mean the same thing. (top-right-bottom-left). If you leave off the last one, then it makes right and left the same. It's more concise and generally recommended to do it that way